app/models/invenio_rdm_record_converter.rb

```diff
@@ -84,20 +84,21 @@ def extra_data
     data
   end
 
-  def owner_info
-    user = User.find_by(username: @generic_file.depositor)
+  def owner_info(depositor)
```

Comment:
Adding the argument makes it easier to test because you can then re-use an instance of InvenioRdmRecordConverter without running all the setup again.

Reply:
For digitalhub at this point it's fine. I think in general this is probably a sign that `InvenioRdmRecordConverter` is lacking an easy creation or that the tests need rearranging.